archive_extractor/zip_archiver.go

```diff
@@ -11,6 +11,17 @@ import (
 )
 
 const fileHeaderSignatureString = "PK\x03\x04"
+const zoneInfoFileHeaderSignatureString = "\x23\x20\x76\x65\x72\x73\x69\x6F\x6E"
+
+type ZoneInfoFileError struct {
+	message string
+}
+
+func (e *ZoneInfoFileError) Error() string {
+	return e.message
+}
+
+var ErrZoneInfoFile = &ZoneInfoFileError{"zone info file found instead of zip"}
```

Comment:
This var isn't needed.
Let's have this error msg as const, and this error can be an empty struct and just return this msg in Error() method.